src/main/cpp/src/config/pb_config.cc

```diff
@@ -215,6 +215,9 @@ void GenomicsDBConfigBase::read_from_PB(const genomicsdb_pb::ExportConfiguration
 void GenomicsDBConfigBase::read_and_initialize_vid_and_callset_mapping_if_available(
     const genomicsdb_pb::ExportConfiguration* export_config) {
   //Callset mapping and vid mapping
+  if (m_vid_mapper.is_initialized()) {
+    return;
+  }
```

Comment:
Is there a scenario where vid_mapper can be initialized before this?
Scratch that - I see the update from loader function call below.
I would recommend moving this check to near line 226:
```C++
if(has_vid_mapping_file()) 
  do_whatever();
else
  if(has_vid_mapping())
    do_whatever();
  else
     if(!m_vid_mapper.is_initialized())
      throw exception();
```
This ensures that any vid mapping defined in the query object has higher priority than the loader config

Reply:
Will do